web/src/components/task/task-sidebar/finish-button/utils.ts

```diff
@@ -1,7 +1,29 @@
+const LISTED_PAGES_LIMIT = 10;
+const SLICE_START_INDEX = 0;
+const SLICE_END_INDEX = LISTED_PAGES_LIMIT - 1;
+
+const getPagesList = (pages: number[]) => pages?.join(', ');
+
+const getAboveLimitPagesCount = (pages: number[]) => pages.length - LISTED_PAGES_LIMIT;
+
+const getVisiblePagesString = (pages: number[]) => getPagesList(getVisiblePagesArray(pages));
+
+const getVisiblePagesArray = (pages: number[]) => pages.slice(SLICE_START_INDEX, SLICE_END_INDEX);
```

Comment:
Let's move this before "getVisiblePagesString" since in the future we may accidentally catch the case when "getVisiblePagesString" is called earlier than "getVisiblePagesArray" is defined.

Reply:
Addressed